tests/projects/test_projects_cli.py

```diff
@@ -217,4 +219,6 @@ def test_mlflow_run():
             cli.run, ["--experiment-id", "51", "--experiment-name", "name blah", "uri"]
         )
         mock_projects.run.assert_not_called()
-        assert "Specify only one of 'experiment-name' or 'experiment-id' options." in result.output
+        # click >= 8.2.0 separates stdout and stderr
+        stderr = result.stderr if Version(click.__version__) >= Version("8.2.0") else result.output
+        assert "Specify only one of 'experiment-name' or 'experiment-id' options." in stderr
```

Comment:
Handle a breaking change in https://github.com/pallets/click/releases/tag/8.2.0.